src/Sulu/Bundle/AdminBundle/Resources/js/stores/ResourceStore/tests/ResourceStore.test.js

```diff
@@ -525,7 +525,7 @@ test('Copy the content from different locale', () => {
     resourceStore.copyFromLocale('de');
 
     expect(ResourceRequester.post)
-        .toBeCalledWith('pages', {}, {action: 'copy-locale', id: 4, locale: 'de', dest: 'en'});
+        .toBeCalledWith('pages', {}, {action: 'copy-locale', id: 4, locale: 'en', dest: 'en', src: 'de'});
```

Comment:
This is a bc break as we are now sending the `dest` locale as locale and this will break existing bundles as the now get a false locale for `copy-locale` action.

Reply:
Other existing bundles are probably broken in the same way as the pages are, so they need fixing anyway. Therefore I would maybe mention that in the `UPGRADE.md`, but I would still not make two requests, since this is bad performance-wise.

Reply:
We still can not `change` the `locale` parameter here, it will break more things as before for this bundles as it will try then to copy from `en` to `en`. The `locale` and `dest` should stay the same and we could add another additional parameter `return-dest=true` or something, else it will break things more as it was done before.

Reply:
I think there are not too many bundles out there relying on that (probably just the ones we've created on our own), therefore I would not go down that road and build a very complicated API.

In addition to that `return-dest` won't work for all cases, since the API also accepts multiple destination locales, and we can only return one. We could keep the `locale` and `dest` query parameters untouched and introduce a `return-locale` variable instead, which decided the locale of the response. But it is harder to grasp what is used as source, and it is inconsistent to me, since the `locale` usually describes the locale of the response. And in order to get rid of the bug the API still has to be touched for the other bundles.

For these reasons I would accept the BC break for the sake of clarity and consistency. It is also documented in the `UPGRADE.md`, so there is a hint for people having implemented something like that. Nevertheless I am willing to change it using a `return-locale` query parameter instead, if @sulu/core-team prefers that.